File: templates/Uwp/_comp/MVVMLight/Project/Activation/DefaultActivationHandler_postaction.cs
//{[{
using Param_RootNamespace.ViewModels;
//}]}
namespace Param_RootNamespace.Activation
{
    internal class DefaultActivationHandler : ActivationHandler<IActivatedEventArgs>
    {
//{[{
        private readonly string _navElement;

        public NavigationServiceEx NavigationService => ViewModelLocator.Current.NavigationService;

        public DefaultActivationHandler(Type navElement)
        {
            _navElement = navElement.FullName;
        }

//}]}
    }
}

File: templates/Uwp/_comp/MVVMLight/Project._VB/Activation/DefaultActivationHandler_postaction.vb
'{[{
Imports Param_RootNamespace.ViewModels
'}]}
Namespace Activation
    Friend Class DefaultActivationHandler
        Inherits ActivationHandler(Of IActivatedEventArgs)
'{[{

        Private ReadOnly _navElement As string

        Private ReadOnly Property NavigationService As NavigationServiceEx
            Get
                Return ViewModelLocator.Current.NavigationService
            End Get
        End Property

        Public Sub New(navElement As Type)
            _navElement = navElement.FullName
        End Sub
'}]}
    End Class
End Namespace
